doc/sources/index.rst

```diff
@@ -0,0 +1,87 @@
+.. _index:
+
+#####################################################
+Intel(R) Extension for Scikit-learn*
+#####################################################
+Intel(R) Extension for Scikit-learn speeds up scikit-learn beyond by providing drop-in patching.
+Acceleration is achieved through the use of the Intel(R) oneAPI Data Analytics Library (oneDAL)
+that allows for fast usage of the framework suited for Data Scientists or Machine Learning users.
+
+Designed for Data Scientists and Framework Designers
+----------------------------------------------------
+Intel(R) Extension for Scikit-learn* was created to give data scientists the easiest way to get performance, 
+when using familiar Scikit-learn package.
+
+Usage
+--------------------
+Intel(R) Extension for Scikit-learn* dynamically patches scikit-learn estimators to use Intel(R) oneAPI Data Analytics Library
+as the underlying solver, while getting the same solution faster.
+
+- It is possible to enable those patches without editing the code of a scikit-learn application by
+  using the following commandline flag::
+
+    python -m sklearnex my_application.py
+
+- Or from your script::
+
+    from sklearnex import patch_sklearn
+    patch_sklearn()
+
+
+For example::
+
+    from sklearnex import patch_sklearn
+    patch_sklearn()
+
+    from sklearn.cluster import KMeans
+    import numpy as np
+    X = np.array([[1, 2], [1, 4], [1, 0],
+                  [10, 2], [10, 4], [10, 0]])
+    kmeans = KMeans(n_clusters=2, random_state=0).fit(X)
+    print(f"kmeans.labels_ = {kmeans.labels_}")
+    pred = kmeans.predict([[0, 0], [12, 3]])
+    print(f"pred = {pred}")
+    print(f"kmeans.cluster_centers_ = {kmeans.cluster_centers_}")
```

Comment:
could you please check that this is rendered in the output with the proper syntax highlighting?

Reply:
Yes, syntax is highlighting correctly
![image](https://user-images.githubusercontent.com/35104773/112964619-2b506680-9151-11eb-9550-cb03b408ea96.png)